integration_test/tds/insert_all_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Ecto.Integration.InsertAllTest do
+  use Ecto.Integration.Case, async: true
+
+  alias Ecto.Integration.TestRepo
+  alias Ecto.Integration.Post
+  import Ecto.Query
+
+  @tag :insert_all
+  test "insert_all" do
+    assert {:ok, %Post{id: id}} = TestRepo.insert(%Post{
+      title: "A generic title"
+    })
+
+    source = from p in Post,
+      select: %{
+        title: fragment("concat(?, ?)", ^"foobar", p.id)
+      }
+
+    expected_title = "foobar#{id}"
+    assert {1, [%Post{title: ^expected_title}]} = TestRepo.insert_all(Post, source, returning: [:id, :title])
```

Comment:
We should add this test to ecto/integration_test/cases/repo.exs. And then define some variations (with conflict_target, without conflict_target, etc). You can see how the current insert_all tests are structured. For convenience testing, you can do this:

    ECTO_PATH=path/to/ecto ECTO_ADAPTER=pg mix test path/to/ecto/integration_test/cases/repo.exs

Reply:
You will also need to use the existing tags to skip some of the tests for some adapters.

Reply:
So the integration tests that are defined in ecto are still only executed when I run the integration tests in ecto_sql?

Reply:
done